modules/javafx.graphics/src/main/java/javafx/scene/robot/Robot.java

```diff
@@ -0,0 +1,362 @@
+     * @param buttons the mouse buttons to click
+     * @throws IllegalStateException if this method is called on a thread
+     * other than the JavaFX Application Thread.
+     */
+    public void mouseClick(MouseButton... buttons) {
+        mousePress(buttons);
+        mouseRelease(buttons);
+    }
+
+    /**
+     * Scrolls the mouse wheel by the specified amount. Positive {@code wheelAmt}
+     * scrolls up whereas negative scrolls down.
+     *
+     * @param wheelAmt the (signed) amount to scroll the wheel
+     * @throws IllegalStateException if this method is called on a thread
+     * other than the JavaFX Application Thread.
+     */
+    public void mouseWheel(int wheelAmt) {
+        peer.mouseWheel(wheelAmt);
+    }
+
+    /**
+     * Returns the {@link Color} of the pixel at the specified screen coordinates of the
+     * primary screen.
+     *
+     * @param x the x coordinate to get the pixel color from
+     * @param y the y coordinate to get the pixel color from
+     * @return the pixel color at the specified screen coordinates
+     * @throws IllegalStateException if this method is called on a thread
+     * other than the JavaFX Application Thread.
+     */
+    public Color getPixelColor(double x, double y) {
+        return peer.getPixelColor(x, y);
+    }
+
+    /**
+     * Returns the {@link Color} of the pixel at the screen coordinates relative to the
+     * primary screen specified by {@code location}.
+     *
+     * @param location the (x,y) coordinates to get the pixel color from
+     * @return the pixel color at the specified screen coordinates
+     * @throws NullPointerException if the given {@code location} is {@literal null}
+     * @throws IllegalStateException if this method is called on a thread
+     * other than the JavaFX Application Thread.
+     */
+    public Color getPixelColor(Point2D location) {
+        Objects.requireNonNull(location);
+        return getPixelColor(location.getX(), location.getY());
+    }
+
+    /**
+     * Returns a {@link WritableImage} containing the specified rectangular area of the
+     * primary screen. If the given {@code image} is {@literal null}, or if the given
+     * {@code image} is too small, a new {@code WritableImage} will be created and returned.
+     * Otherwise, the given {@code image} is re-used.
+     * <p>
+     * If the {@code scaleToFit} argument is {@literal false}, the returned
+     * {@code Image} object dimensions may differ from the requested {@code width}
+     * and {@code height} depending on how many physical pixels the area occupies
+     * on the screen. For example, in HiDPI mode on the Mac (aka Retina display) the
+     * pixels are doubled, and thus a screen capture of an area of size (10x10) pixels
+     * will result in an {@code Image} with dimensions (20x20). Calling code should
+     * use the returned images's {@link Image#getWidth() and {@link Image#getHeight()
```

Comment:
You are missing the closing curly brace for the two `@link` tags. This causes `gradle javadoc` to fail.